tests/test_config.py

```diff
@@ -10,6 +11,16 @@
 
 
 class ConfigTest(unittest.TestCase):
+
+    def _assertDatetimeAlmostEqual(self, s1, s2):
+        """Assert that two date strings are almost equal."""
+        s1 = datetime.strptime(s1, "%a, %d %b %Y %H:%M:%S %z")
+        s2 = datetime.strptime(s2, "%a, %d %b %Y %H:%M:%S %z")
+        return self.assertTrue(
+            abs(s1 - s2) < timedelta(seconds=1),
```

Comment:
This delta value could be a third optional parameter

Reply:
Good suggestion. Implemented.